ocs_ci/ocs/utils.py

```diff
@@ -890,6 +892,45 @@ def collect_noobaa_db_dump(log_dir_path):
     )
 
 
+def save_pods_logs_to_file(pod_patterns_containers, pattern_to_log=None):
+    """
+    Save live logs of requested pods
+
+    Args:
+        pod_patterns_containers (dict): The patterns of the pods that
+            their logs should be saved as keys and the container names as values
+        pattern_to_log (str): A pattern in the logs to look for and log
+
+    """
+    log_dir_path = os.path.join(
+        os.path.expanduser(ocsci_config.RUN["log_dir"]),
+        f"live_logs_{ocsci_config.RUN['run_id']}",
+    )
+    pods_containers = dict()
+    for pattern, containers in pod_patterns_containers.items():
+        pod_names = get_pod_name_by_pattern(pattern=pattern)
+        for pod_name in pod_names:
+            pods_containers.update({pod_name: containers})
+
+    now = datetime.datetime.now()
+    dt_string = now.strftime("%d/%m/%Y %H:%M:%S")
+    live_logs_dir_path = os.path.join(log_dir_path, f"live_logs_{dt_string}")
```

Comment:
and one more thing for improvement. live_logs_31/ can you replace it maybe with the name of the current test case?

There doesn't have to be live_logs in the name again if you already have it in parent folder live_logs_1617148726 .

The path now looks like:
`live_logs_1617148726/live_logs_31/03/2021 00:09:34/`


So I think it should be something like:
`live_logs_1617148726/tc_name/210331_00:09:34/`

Not sure if we need to have date and too many folders

Reply:
done